test/test_nn.py

```diff
@@ -4463,6 +4463,18 @@ def pad(tensor, length):
         with self.assertRaisesRegex(RuntimeError, 'You can pass `enforce_sorted=False`'):
             packed = rnn_utils.pack_padded_sequence(torch.randn(3, 3), [1, 3, 2])
 
+    @unittest.skipIf(not TEST_CUDA, 'CUDA not available')
+    def test_pack_padded_sequence_default_cuda_tensor(self):
+        # Set default tensor type to cuda.FloatTensor, as this has triggered the error
+        torch.set_default_tensor_type(torch.cuda.FloatTensor)
+        # Run all tests
+        try:
+            self.test_pack_sequence()
+            self.test_pack_padded_sequence()
+        # Reset default tensor type to FloatTensor
+        finally:
+            torch.set_default_tensor_type(torch.FloatTensor)
```

Comment:
This might be wrong, because I know we set the default tensor type to DoubleTensor for many of our tests. Better to read out what the current default tensor type is and then restore it at the end.

Reply:
I was thinking about this approach, but wasn't sure if there is a designated method for it.
A workaround might be:
```python
default_type = torch.Tensor(1).type()
```
However, I feel it looks a bit hacky.
What do you think?